### ISSUE TITLE
DietPi Software | Mopidy fresh installation playback but no sound

### ISSUE THREAD

**Issue by arkhub** [MichaIng/DietPi#2536]
# Fix: https://github.com/Fourdee/DietPi/issues/2536#issuecomment-462433840

---

#### Required Information
- DietPi version : 6.21.1
- Distro version : stretch
- Kernel version : Linux <devicename> 4.14.79-v7+ #1159 SMP Sun Nov 4 17:50:20 GMT 2018 armv7l GNU/Linux
- SBC device : RPi 3 Model B+ (armv7l)
- Power supply used : 5V 3A
- SDcard used : 32 gb microsd card

#### Additional Information (if applicable)
- Software title : mopidy
- Was the software title installed freshly or updated/migrated? fresh installation of mopidy (but dietpi software was updated from 6.19 to 6.20 and 6.21 before)
- Can this issue be replicated on a fresh installation of DietPi? I think yes

#### Steps to reproduce

1. install mopidy from dietpi-software
2. open browser at http://raspberry-pi-address:6680/musicbox_webclient/
3. try to play sample audio

#### Expected behaviour

- sound should be played

#### Actual behaviour

- no sound at all but playback running

#### Extra details

- try sound test using ```gst-launch-1.0 audiotestsrc ! audioresample ! alsasink``` produce sounds
- stopping the mopidy services and then execute ```mopidyctl -v``` command makes the playback works (there's sound!)
- exiting ```mopidyctl -v``` and starting the services again using ```service mopidy start``` and the sounds dissapears again.

**Comment by MichaIng** [comment]
@arkhub 
Thanks for your report.

We will investigate.

**Comment by Fourdee** [comment]
Confirmed bug, no ALSA stream active, will investigate:
```
Feb 11 18:09:42 DietPi mopidy[1045]: WARNING  Failed looking up file:///mnt/diet
pi_userdata/Music/%5BRadio%20Streams%5D.m3u8: gst-stream-error-quark: Stream con
tains no data. (4)
Feb 11 18:09:43 DietPi mopidy[1045]: WARNING  Failed looking up file:///mnt/diet
pi_userdata/Music/%5BRadio%20Streams%5D.m3u8: gst-stream-error-quark: Stream con
tains no data. (4)
Feb 11 18:09:44 DietPi mopidy[1045]: WARNING  GStreamer warning: gst-resource-er
ror-quark: Could not open audio device for playback. (6)
```

**Comment by Fourdee** [comment]
## Fix:
```
dietpi-services stop
usermod -a -G dietpi,audio -d $G_FP_DIETPI_USERDATA/mopidy mopidy
dietpi-services start
```

**Comment by Fourdee** [comment]
@arkhub 

Many thanks for reporting this issue 👍 

You can apply the fix now with: https://github.com/Fourdee/DietPi/issues/2536#issuecomment-462433840

Also patched for v6.22.

Completed
```
 DietPi-JustBoom
─────────────────────────────────────────────────────
 Mode: ALSA Output Info

access: RW_INTERLEAVED
format: S16_LE
subformat: STD
channels: 2
rate: 44100 (44100/1)
period_size: 440
buffer_size: 8820
```
